assembly/pom.xml

```diff
@@ -220,6 +222,7 @@
               <arguments>
                 <argument>dependency:copy-dependencies</argument>
                 <argument>-DoutputDirectory=${basedir}/target/scala-${scala.binary.version}/jars/connect-repl</argument>
+                <argument>-DincludeScope=runtime</argument>
```

Comment:
IIRC, maven-dependency-plugin also support `<includeScope>runtime</includeScope>`

Reply:
Yeah, the problem is `--file` which is a Maven CLI option so can't replace this.